### PR TITLE
add s3 bucket, policies

### PR DESCRIPTION
**Pull request by david-wirelab**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -5,6 +5,9 @@ variable "data_ingest_cidr_block" {}
 variable "peering_cidr_block" {}
 variable "az" {}
 variable "data_landing_bucket" {}
+variable "s3_bucket_name" {}
+variable "s3_bucket_acl" {}
```
Comment (chrisns) [inline variables.tf]: won't this always be private? does it need to be a var?

File: variables.tf [hunk 2]
```diff
@@ -5,6 +5,9 @@ variable "data_ingest_cidr_block" {}
 variable "peering_cidr_block" {}
 variable "az" {}
 variable "data_landing_bucket" {}
+variable "s3_bucket_name" {}
```
Comment (chrisns) [inline variables.tf]: can't you auto generate a random name?

File: variables.tf [hunk 3]
```diff
@@ -5,6 +5,9 @@ variable "data_ingest_cidr_block" {}
...
+variable "s3_bucket_acl" {}
+variable "region" {}
```
Comment (chrisns) [inline variables.tf]: can't you get this from the account?

File: variables.tf (hunk 2 above)
Comment (david-wirelab) [reply variables.tf]: sure we can but perhaps we wan't an easily readable name?

File: variables.tf (hunk 1 above)
Comment (david-wirelab) [reply variables.tf]: it may not necessary be set to private: https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl

Comment (chrisns) [reply variables.tf]: so we can remove the param entirely?

File: variables.tf (hunk 3 above)
Comment (david-wirelab) [reply variables.tf]: agree will change this.

File: variables.tf (hunk 2 above)
Comment (chrisns) [reply variables.tf]: prefix it with something meaningful, see how we do ssm docs https://github.com/UKHomeOffice/dq-tf-ad/blob/master/main.tf#L185

File: variables.tf (hunk 1 above)
Comment (david-wirelab) [reply variables.tf]: actually for this bucket it is not required. removing it.

**Pull request merged**